alis.sh

```diff
@@ -419,7 +424,7 @@ function partition() {
         if [ "$BIOS_TYPE" == "uefi" ]; then
             parted -s "$DEVICE" "$PARTITION_PARTED_UEFI"
             if [ -n "$LUKS_PASSWORD" ]; then
-                sgdisk -t="$PARTITION_ROOT_NUMBER":8309 "$DEVICE"
+                sgdisk -t="$PARTITION_ROOT_NUMBER":8304 "$DEVICE"
```

Comment:
What is the reason for this change?

Reply:
Otherwise systemd-gpt-auto-generator is not trying to decrypt the partition.

I am not entirely sure but as far as I understood, according to "The Discoverable Partitions Specification" (see https://uapi-group.org/specifications/specs/discoverable_partitions_specification/), the underlying partition GUID after decryption needs to be specified, which here is Linux x86-64 root (/).